src/fontra/client/core/vector.js

```diff
@@ -7,7 +7,11 @@ export function subVectors(vectorA, vectorB) {
 }
 
 export function mulVector(vector, scalar) {
```

Comment:
Instead of modifying this function, let's add a new one vor vector * vector called `mulVectorVector()`.

Reply:
Additionally, we could consider renaming the original function to `mulVectorScalar()`.

Reply:
Ok. That makes sense